### PR TITLE
Includes last invoice date for cabinet

### PR DESCRIPTION
**Pull request by javimudi**
Hi, team,

We're proposing this attribute to handle invoices-oriented grouping for energy counters.

Since invoices issuing date may differ over time, tracking dates for invoices seems a good solution.

Regards

### CODE REVIEW

File: StreetLighting/StreetlightControlCabinet/doc/spec.md [hunk 1]
```diff
@@ -75,7 +75,11 @@ responsible, district, neighbourhood, etc.
 + `dateLastProgramming` : Date at which there was a programming operation over the cabinet.
     + Attribute Type: [Date](http://schema.org/DateTime)
     + Optional
-    
+
++ `dateLastInvoice` : Date at which the last invoice for the cabinet was issued.
```
Comment (jmcanterafonseca) [inline StreetLighting/StreetlightControlCabinet/doc/spec.md]: we need to be more clear with the description

the last invoice corresponding to the energy consumed by the circuits controlled by this cabinet

**Comment by jmcanterafonseca** [comment]
then if it includes information of other urban things it is not a
StreetlightControlCabinet ...

2016-10-10 14:18 GMT+02:00 Javi Muñoz <notifications@github.com>:

**Comment by javimudi** [comment]
Definitely that's correct with most of scenarios, with one electrical
measurement per ControlCabinet and no other uncontrolled circuits. Under
this schema, no secondary nor unmanaged circuits are allowed, and both
situations could exist.

A ControlCabinet is indeed behaving both like a controlling and like a
measures reading device, so a more complete (and way more complicated)
model might consider including splitted functional models for control and
measurement.



2016-10-10 14:21 GMT+02:00 Jose M. Cantera <notifications@github.com>:

**Comment by jmcanterafonseca** [comment]
I can understand and thanks for the detailed explanation. please could you update the PR to deal with the latest discussions and for being as precise as possible? 

thanks!

**Comment by albertoabellagarcia** [comment]
I am the new manager of the data models initiative. I know that this initiative is rather old. But keep willing to receive your contributions.
Now all the datamodels are available here (https://github.com/smart-data-models/) and this data model specifcally here (https://github.com/smart-data-models/dataModel.Energy/tree/master/ThreePhaseAcMeasurement). could you move this pull request there?
Reach  me at alberto . abella @ fiware . org

**Pull request closed without merging**